testing/test_assertion.py

```diff
@@ -776,6 +776,17 @@ def test_mojibake(self) -> None:
         msg = "\n".join(expl)
         assert msg
 
+    def test_nfc_nfd_same_string(self) -> None:
+        # issue 3426
+        left = "hyv\xe4"
+        right = "hyva\u0308"
+        expl = callequal(left, right)
+        assert expl == [
+            "\"'hyv\\\\xe4'\" == \"'hyva\\\\u0308'\"",
```

Comment:
Using a raw-string with single-quotes should make this easier to manage - `r'"hyv\xe4" == "hyva\u0308"'`